src/components/Snippet.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { Controlled as CodeMirror } from 'react-codemirror2';
+
+import 'codemirror/lib/codemirror.css';
+
+import Title from './common/Title';
+import Input from './common/Input';
+
+import '../styles/Snippet.styl';
+
+class Snippet extends Component {
+  constructor() {
+    super();
+    this.state = { isShowEmbed: false };
+    this.toggleEmbed = this.toggleEmbed.bind(this);
+  }
```

Comment:
(minor) I think it'd be great to separate methods from each other with an empty line.